nx/include/switch/services/set.h

```diff
@@ -954,6 +1903,92 @@ Result setsysGetPctlReadyFlag(bool *out);
  */
 Result setsysSetPctlReadyFlag(bool flag);
 
+/**
+ * @brief GetAnalogStickUserCalibrationL
+ * @note Only available on [8.1.1+].
+ * @param[out] out \ref SetSysAnalogStickUserCalibration
+ */
+Result setsysGetAnalogStickUserCalibrationL(SetSysAnalogStickUserCalibration *out);
+
+/**
+ * @brief SetAnalogStickUserCalibrationL
+ * @note Only available on [8.1.1+].
+ * @param[in] calibration \ref SetSysAnalogStickUserCalibration
+ */
+Result setsysSetAnalogStickUserCalibrationL(const SetSysAnalogStickUserCalibration *calibration);
+
+/**
+ * @brief GetAnalogStickUserCalibrationR
+ * @note Only available on [8.1.1+].
+ * @param[out] out \ref SetSysAnalogStickUserCalibration
+ */
+Result setsysGetAnalogStickUserCalibrationR(SetSysAnalogStickUserCalibration *out);
+
+/**
+ * @brief SetAnalogStickUserCalibrationR
+ * @note Only available on [8.1.1+].
+ * @param[in] calibration \ref SetSysAnalogStickUserCalibration
+ */
+Result setsysSetAnalogStickUserCalibrationR(const SetSysAnalogStickUserCalibration *calibration) ;
```

Comment:
Remove extra space before `;`.

Reply:
also caught one at setsysSetThemeId and setsysGetPtmBatteryLot